client/my-sites/domains/domain-management/domain-overview-pane/index.tsx

```diff
@@ -73,12 +73,15 @@ const DomainOverviewPane = ( {
 
 		return (
 			<>
-				<Button onClick={ closeSitePreviewPane } className="item-preview__close-preview-button">
+				<Button
+					onClick={ closeSitePreviewPane }
+					className="button item-preview__close-preview-button"
+				>
 					{ __( 'Close' ) }
 				</Button>
 				<Button
 					variant="primary"
-					className="item-preview__admin-button"
+					className="button item-preview__admin-button"
```

Comment:
Very valid question. Actually, the `button` class adds a bunch of styles necessary to make the button look good, especially to match the buttons on the wordpress.com/sites page. Otherwise, we'd have to duplicate the styles. So I chose to re-use the class here instead.